python/common/org/python/types/Function.java

```diff
@@ -19,6 +19,14 @@
     java.util.Map<java.lang.String, org.python.Object> default_kwargs;
     org.python.types.Closure closure;
 
+    public static java.lang.String protect_reserved_words(java.lang.String arg) {
```

Comment:
Hey!
I tried out running the code from this PR, and everything seems fine.
About the code, I was trying to figure out where this method is being used, but couldn't find -- could you clarify?
Thanks!